synapse/tools/genpkg.py

```diff
@@ -198,6 +198,11 @@ def loadPkgProto(path, opticdir=None, no_docs=False, readonly=False):
             with s_common.genfile(cmd_path) as fd:
                 cmd['storm'] = fd.read().decode()
 
+    for gdef in pkgdef.get('graphs', ()):
+        gdef['iden'] = s_common.guid((pkgname, gdef.get('name')))
```

Comment:
Did you consider having this be hard-coded like the meta source iden so that a power-up could easily get() the graph it loaded?

Reply:
Yeah, I was trying to avoid allowing specifying an arbitrary iden to prevent collision issues. Generating the guid from the package+graph name seemed like a decent compromise since it would still be predictable and you could just calculate it in a module or drop it in a modconf.

Reply:
we could also do something like a 'byname' api.

Reply:
Do we persist these package graph idens anywhere other than the runtime `Cortex.pkggraphs` value?

Reply:
No, are there other places that they should be stored as well?